Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3366,8 +3366,8 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
     auto pagePoint = convertDown(CoordinateSpace::PDFDocumentLayout, CoordinateSpace::PDFPage, documentPoint, *pageIndex);
     RetainPtr page = m_documentLayout.pageAtIndex(*pageIndex);
     RetainPtr wordSelection = [page selectionForWordAtPoint:pagePoint];
-    if (!wordSelection)
-        return { { }, nil };
+    if (!wordSelection || [wordSelection isEmpty])
+        return { { }, wordSelection ?: nil };
```

Comment:
Wouldn't `wordSelection ?: nil` just be equivalent to `wordSelection`?

```suggestion
        return { { }, wordSelection };
```

Reply:
😅 of course